stack.yaml

```diff
@@ -102,21 +104,24 @@ extra-deps:
   commit: 26d35ad52fe9ade3391532dbfeb2f416f07650bc
 
 - git: https://github.com/input-output-hk/cardano-prelude
-  commit: 00487726c4bc21b4744e59d913334ebfeac7d68e
+  commit: fe76ec64f6b45259cc407a6d840dad79ee6063b6
   subdirs:
     - .
     - test
 
 - git: https://github.com/input-output-hk/ouroboros-network
-  commit: 398004e1403367cc2a25c639eb6349d473e51b2d
+  commit: a85bd4751ca5c81c0507482848358980814e9ca3
   subdirs:
     - io-sim
     - io-sim-classes
     - network-mux
     - ouroboros-network
     - ouroboros-consensus
+    - ouroboros-consensus-byron
     - typed-protocols
-    - typed-protocols-cbor
+    - typed-protocols-examples
+    - ouroboros-network-framework
+    - ouroboros-network-testing
 
 - git: https://github.com/input-output-hk/ouroboros-network
   commit: f12e7c2168fe3bc4e236898a6e0207479cda3302
```

Comment:
@rvl  @Anviking  please take notice that we used this trick (repeating ouroboros-network with newer commit) to have implementation of ntp-client. I bet now it can be erased and `ntp-client` can be put in subdirs above. It would be worth checking this

Reply:
Thanks @paweljakubas - I have addressed this in de807ece5459ec002175f292e5de9b0f9770e821.